transport_analysis/velocityautocorr.py

```diff
@@ -262,3 +261,68 @@ def plot_vacf(self, start=0, stop=0, step=1):
             self.times[start:stop:step],
             self.results.timeseries[start:stop:step],
         )
+
+    def sd(self, start=0, stop=0, step=1):
```

Comment:
This is a green Kubo integral of the VACF so we should use the proper name.

Reply:
Also will need citations.

Reply:
Does `sd_gk()` sound good? I feel that `self-diffusivity_greenkubo()` is too long. I thought we were doing citations after we finish the bulk of the implementations? Or would it be better to do them now?

Reply:
`sd_gk` sounds good to me. Either is fine on citation, up to you, just make a note or issue.

Reply:
Left a note in Issue #20.